test/parallel/test-https-timeout.js

```diff
@@ -30,8 +31,8 @@ const https = require('https');
 const fs = require('fs');
```

Comment:
@	mikemfleming - on a second look, the requiring of fs module is not used anywhere in the test, and I guess is likely to invite linter error. Will you remove this please?

Reply:
Just removed it

Reply:
thanks!